### PR TITLE
IS-1106: Version management abiding by the rules of PEP

### PR DESCRIPTION
**Pull request by soobokjin**
## Summary
Record version at __init__ and Remove VERSION file. that is the standard ways of versioning
- As importing external lib on __init__.py, It is impossible to build a wheel before installing all requirements. 
- To avoid this problem, Define __version__.py and extract version info from it.
- "requests" lib has the same problem and solve it using the same way

Remove deprecated scripts.
- build.sh
- deploy.sh

### CODE REVIEW

**Comment by yakkle** [comment]
could you add referenced link of PEP?

**Comment by soobokjin** [comment]
@yakkle  https://www.python.org/dev/peps/pep-0396/
I have interpreted about this PEP on Jira.

File: setup.py [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+CURRENT_PYTHON = sys.version_info[:3]
+REQUIRED_PYTHON = (3, 6, 5)
+
+if CURRENT_PYTHON < REQUIRED_PYTHON:
+    sys.stderr.write(
+        f"Unsupported Python version* This version of Icon Service requires\n"
+        f"Python {'.'.join([str(v) for v in REQUIRED_PYTHON])}, "
+        f"but you're trying to install it on Python {'.'.join([str(v) for v in CURRENT_PYTHON])}. ")
+    sys.exit(1)
+
```
Comment (yakkle) [inline setup.py:16]: I recommend to use `python_requires`.
https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

Comment (soobokjin) [reply setup.py]: Thank you for providing better ways! I have applied tpython_requires.

**Pull request merged**